spec/javascripts/AccessTokens/components/ExpirationDatePicker.spec.tsx

```diff
@@ -67,13 +77,12 @@ describe('select a period', () => {
 
   it('should update hidden input value to the correct timestamp', () => {
     const wrapper = mountWrapper()
-    const targetDate = new Date(new Date().getTime() + 1000 * 60 * 60 * 24 * targetItem.period)
-    const expectedValue = targetDate.toISOString().replace(msExp, 'Z')
+    const targetDate = futureDateInNDays(targetItem.period)
 
     selectItem(wrapper, targetItem)
-    const value = (wrapper.find(`input#${defaultProps.id}`).prop('value') as string).replace(msExp, 'Z')
+    const value = new Date(wrapper.find(`input#${defaultProps.id}`).prop('value') as string)
 
-    expect(value).toBe(expectedValue)
+    expect(value).toBeWithinSecondsFrom(targetDate)
```

Comment:
By default, it will be 1 second. I think it should always be enough, I think it will always potentially be just a few milliseconds difference.

But if we need, we can compare that two dates are within e.g. 5 seconds by
```
expect(value).toBeWithinSecondsFrom(targetDate, 5)
```